recipe/meta.yaml

```diff
@@ -22,6 +23,7 @@ requirements:
     - zlib  # [not win]
   host:
     - libcurl  # [not win]
+    - libgit2 >=1.7.2,<1.8.0
```

Comment:
why is this here? is it not compatible with 1.8.0?

Reply:
https://github.com/rust-lang/git2-rs/pull/1032/files

it seems it just isn't compatible yet.

Reply:
Truthfully, for me this is the larger issue given that libgit2 has already moved to 1.8 conda-forge wide and this might cause some conflicts.